packages/expo-updates/android/src/main/java/expo/modules/updates/db/dao/UpdateDao.java

```diff
@@ -95,6 +96,11 @@ public void markUpdateFinished(UpdateEntity update) {
     markUpdateFinished(update, false);
   }
 
+  public void markUpdateAccessed(UpdateEntity update) {
+    update.lastAccessed = new Date();
+    _updateUpdate(update);
```

Comment:
@jkhales - replicating my deleted response to your comment here. All the abstract methods annotated with things like `@Query`/`@Update`/`@Insert`/`@Delete` are implemented in Room's autogenerated implementation of this abstract class.

Reply:
thank you!